### ISSUE TITLE
EventFahrplan only shows one event when there is an event scheduled for different times (Pretalx) ?

### ISSUE THREAD

**Issue by NerdyProjects** [EventFahrplan/EventFahrplan#496]
We use pretalx for event scheduling and have a session being repeated on multiple times in the same pretalx event/session.

The pretalx export uses the same `id` for all these sessions but assigns different `guid`s. EventFahrplan only shows the last of these events, missing all the others.

I guess this might be related to #319 ?

By reading some issues, to me it sounds also unclear how sessions with multiple times should be handled:
- same ID or different per time?
- same GUID or different per time?

Logically, the GUID should be unique, but there should be a possibility to link different instances of the same session together, although I don't expect this from EventFahrplan (different Feature request).

Demo Data: https://workshops.acronia.de/acronia-2022/talk/TEQ8ZB/ (Session with multiple timeslots)
https://workshops.acronia.de/acronia-2022/schedule/export/schedule.xml

EventFahrplan only shows the most recent event, 2022-08-21, 09:00–10:00, Black Forest

**Comment by johnjohndoe** [comment]
Thank you for reporting this in detail and your analysis. What you stated is correct. At this point the app has no concept of repeated sessions. It also depends on the ID being unique as you stated correctly. As the migration to GUID is unfinished there is no option to rely on these.

For now you could (1) make sure to output unique IDs from Pretalx or (2) try out the work in progress state of the GUID branch (although I have no idea of what state it is right now as I have not touched it for a while).

**Comment by johnjohndoe** [comment]
@NerdyProjects Did you use the app for your event, finally? Is it published anywhere?

**Comment by NerdyProjects** [comment]
Yes, you can find it in playstore, "Acronia 2022". I think it wasn't widely promoted and we did not fix the issue in any way for this event. I only helped out a friend configuring and compiling it :-)

**Comment by johnjohndoe** [comment]
Ah, cool. I found [it](https://play.google.com/store/apps/details?id=info.metadude.android.acronia.schedule). :+1: 
Your are welcome to report your experience with the setup and other findings to improve the project. You can also use [GitHub discussions](https://github.com/EventFahrplan/EventFahrplan/discussions) for this if you like.

**Comment by navlisilvan** [comment]
Hello, the issue is still there, isn't it?
Is there a workaround to get around this?

**Comment by johnjohndoe** [comment]
Yes, nobody worked on the issue by now. I am not aware of a workaround other than what was stated above.

**Comment by johnjohndoe** [comment]
Please note that I now documented **limitations** of the project [here](https://github.com/EventFahrplan/EventFahrplan/blob/master/docs/LIMITATIONS.md).

**Comment by johnjohndoe** [comment]
🔗 https://github.com/pretalx/pretalx/issues/2248
